### PR TITLE
Feature/skills section

### PR DESCRIPTION
**Pull request by olek-byte**

### CODE REVIEW

**Comment by olek-byte** [comment]
Hello

File: css/_global.css [hunk 1]
```diff
@@ -44,4 +44,22 @@ a {
...
     max-width: 100%;
+}
+
+button {
+  padding: 0;
+  border: none;
+  font: inherit;
+  color: inherit;
+  background-color: transparent;
+  cursor: pointer;
```
Comment (ArtyomKoroteev) [inline css/_global.css]: I recommend you to add default hover state for button. For example:
button::hover {
 opacity: 0.8;
}

File: css/components/_footer.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
+.footer{
+    background-color: #344b8e;
```
Comment (ArtyomKoroteev) [inline css/components/_footer.css]: should be variable

File: css/components/_form.css [hunk 3]
```diff
@@ -0,0 +1,94 @@
+.form {
+    padding: 116px 0;
+    background-color: #3c5499;
```
Comment (ArtyomKoroteev) [inline css/components/_form.css]: You have to put valut into variable across this file

File: css/components/_form.css [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    max-width: 100%;
+}
+
+.form__label:not(:last-child) {
+    margin-right: 33px;
+}
+
+.form__input, .form__textarea {
+    padding: 20px 22px;
+    background-color: #273a71;
```
Comment (ArtyomKoroteev) [inline css/components/_form.css]: same

File: css/components/_form.css [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    border-radius: 6px;
+}
+
+.form__input {
+    width: 414px;
+    height: 56px;
+}
+
+.form__input::placeholder, .form__textarea::placeholder {
+  color: #fff;
```
Comment (ArtyomKoroteev) [inline css/components/_form.css]: same

File: css/components/_form.css [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+.form__btn {
+    background-color: #30bae7;
+}
+
+.form__btn:hover {
+   background-color: #273a71;
```
Comment (ArtyomKoroteev) [inline css/components/_form.css]: same

File: css/components/_form.css [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+@media (max-width: 992px) {
+    .form {
+        padding: 60px 0;
+    }
+
+    .form__input, .form__textarea {
+        display: block;
+        width: 600px;
```
Comment (ArtyomKoroteev) [inline css/components/_form.css]: better use max-width and width 100%. Be careful with width

File: css/components/_form.css [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+   background-color: #273a71;
+   transition: color 200ms;
+}
+
+.form__textarea-box {
+    margin-bottom: 60px;
+}
+
+.form__textarea {
+    height: 200px;
```
Comment (ArtyomKoroteev) [inline css/components/_form.css]: the same thing with height

File: css/components/_team.css [hunk 9]
```diff
@@ -20,47 +14,47 @@
...
 .team__text {
+    height: 120px;
+    margin-bottom: 34px;
     text-align: center;
     font-size: 16px;
     line-height: 24px;
-    margin-bottom: 34px;
 }
 
 .team-social__inner {
```
Comment (ArtyomKoroteev) [inline css/components/_team.css:60]: empty block, remove it

File: css/components/_team.css [hunk 10]
```diff
@@ -73,23 +67,19 @@
...
 }
 
-/* .icon {
-    fill: #fff;
-} */
-
 .team-social__link:hover {
-    fill: #77cfec;
     background-color: #878787;
+    fill: #77cfec;
```
Comment (ArtyomKoroteev) [inline css/components/_team.css]: put color into variable

File: css/components/_team.css [hunk 11]
```diff
@@ -128,7 +117,6 @@
     .team__heading {
         line-height: 40px;
     }
-
 }
 
 @media (max-width: 480px) {
```
Comment (ArtyomKoroteev) [inline css/components/_team.css:122]: empty block

File: js/modules/testModule.js [hunk 12]
```diff
@@ -0,0 +1,19 @@
+export const testModule = (() => {
```
Comment (ArtyomKoroteev) [inline js/modules/testModule.js]: remove this file and import for it

File: js/modules/smoothScrollModule.js [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+    for (const link of links) {
+        link.addEventListener('click', clickHandler);
+    }
+
+    function clickHandler(e) {
+        e.preventDefault();
+        const href = this.getAttribute('href');
+        const offsetTop = document.querySelector(href).offsetTop - 170;
+
+        listActive.classList.remove('header__list--active');
```
Comment (ArtyomKoroteev) [inline js/modules/smoothScrollModule.js:20]: If you add, remove or toggle class, I recommend you put the value of class into variable in such notation:
const HEADER_LIST_ACTIVE_CLASS = 'header__list--active';

File: js/modules/smoothScrollModule.js [hunk 14]
```diff
@@ -0,0 +1,34 @@
...
+        burger.classList.remove('burger--click');
```
Comment (ArtyomKoroteev) [inline js/modules/smoothScrollModule.js:21]: the same

File: js/modules/smoothScrollModule.js [hunk 15]
```diff
@@ -0,0 +1,34 @@
+export const smoothScrollModule = (() => {
+    const listActive = document.querySelector('.header__list');
+    const burger = document.querySelector('.burger');
+    const header = document.querySelector('.header');
+    const links = document.querySelectorAll('.header__link, .header__logo');
+
+    for (const link of links) {
```
Comment (ArtyomKoroteev) [inline js/modules/smoothScrollModule.js]: Better use forEach, for example.
 links.forEach((link) => {
        
});

File: js/modules/pageScrollModule.js [hunk 16]
```diff
@@ -0,0 +1,24 @@
+export const pageScrollModule = (() => {
+    const header = document.querySelector('.header');
+
+    window.onscroll = function () {
```
Comment (ArtyomKoroteev) [inline js/modules/pageScrollModule.js:5]: Use addEventListener

Pack event into separate function

File: js/modules/pageScrollModule.js [hunk 17]
```diff
@@ -0,0 +1,24 @@
...
+        pageScroll();
+    };
+
+    function pageScroll() {
+        let sticky = header.offsetTop;
+        if (window.pageYOffset > sticky) {
+            header.classList.add('sticky');
```
Comment (ArtyomKoroteev) [inline js/modules/pageScrollModule.js:12]: 'sticky' have to be a variable

File: index.html [hunk 18]
```diff
@@ -312,10 +308,64 @@ <h2 class="team__heading heading">Meet Our Beautiful Team</h2>
             </div>
 
         </section>
-    </main>
 
+        <section class="form" id="form">
+            <div class="container">
+                <h2 class="form__heading heading">Get in touch</h2>
+                <p class="form__info info">1600 Pennsylvania Ave NW, Washington, DC 20500, United States of America.
+                    Tel: (202) 456-1111
```
Comment (ArtyomKoroteev) [inline index.html]: pack telephone into `<a href="tel:+6494461709">61709</a>`

File: js/modules/headerModule.js [hunk 19]
```diff
@@ -0,0 +1,21 @@
+export const headerModule = (() => {
+    const listActive = document.querySelector('.header__list');
+    const burger = document.querySelector('.burger');
+
+    function headerMenu() {
+        burger.classList.toggle('burger--click');
```
Comment (ArtyomKoroteev) [inline js/modules/headerModule.js:6]: toggle classes have to be variables

**Review by ArtyomKoroteev — CHANGES_REQUESTED** [review]

**Pull request merged**